### ISSUE TITLE
Loot? Loot! LOOOOOOOT!!!!!

### ISSUE THREAD

**Issue by MrCompost** [Angry-Pixel/The-Betweenlands#169]
Armor of Legends set /Sword of Legends - This is basically THE armour you want to get, it's from defeating the Prime Wight in the final boss battle. It should be twice as good as Valonite in terms of protection and damage, and it should also have a +50% or so boost on all other abilities (jumping, speed, swimming, feather falling, fire damage etc.). Most importantly, they never break, and if you die they remain in your inventory instead of being dropped. 
Skull Mask - Same stats as bone helm, though when worn Wight's are not aggravated when you get near.
Swift Pick - Mines things instantly like creative, has durability of valonite.
Tribal Pants - Just a silly item, only have the same stats as leather armor.
Tribal Spear - Has stats of iron. When right clicked it is pulled back like a bow and then fired (though there isn't an animation for the texture). Can be picked back up. Throwing it uses it up a bit.
Wight's Bane - Kills all Wights in one hit, but against other mobs its just about as good as weedwood.
Elder Staff - This one probably needs it's own issue but I'll put it here anyway. When you right click Cragrock with an Elder Staff, it turns into Enchanted Cragrock which sets your spawn point and a message comes up saying "Your spawn point has been set." If you right click another Cragrock block after, the existing Enchanted Cragrock reverts to regular cragrock and your spawn is set again.
Ring of Power - When in the item bar, your defence and attack get a +50%  boost. It gets used up over time when in the item bar, and lasts half an hour or so overall.
Shadow Staff - Allows you to click and drag mobs around and throw them. Has a cool particle effect that extends from the player and surrounds the mob.
Whip - Basically like a fishing rod but it doesn't fish =P
Explorer's Hat - Just looks cool, same stats as leather armor.
Voodoo Doll - When right clicked, all mobs within a 5x5 area take 20 hearts worth of damage.

**Comment by imdutch21** [comment]
Voodoo Doll -done
Ring of Power -done
Wight's Bane -done
Swift Pick -done
Skull Mask -done
^check these if they function correctly please

**Comment by MrCompost** [comment]
Ok, got some changes and additions to what we have so far.

Voodoo Doll - I think I changed my mind about the usage thing. Imo 5 uses would be much better, as well as a larger range. If there arent any mobs within range, I think that instead of using it up a bit, it should say something like "There are no mobs within range."

Ring of Power - Works well as it is, but it could also do with a couple of messages so the player knows what it is doing. So, when you have it in your hotbar (or inventory, whichever one it is), it could say "You feel the power of the ring making you stronger...", and when you remove it it could then say "You feel the power of the ring leaving you...".

Wight's Bane - Just give it more uses, probably as much as a gold tool.

Swift Pick - Make it faster, FASTER!!!! Almost instant in fact, I imaged it to be almost like creative. I don't know how many uses it has at the moment, but I think something similar to iron would be suitable.

Skull Mask - Works perfectly, just needs to use the overlay when worn and the mask armour texture.
